website/source/api/acl/policies.html.md

```diff
@@ -26,10 +24,10 @@ This endpoint creates a new ACL policy.
 | `PUT`  | `/acl/policy`                | `application/json`         |
 
 The table below shows this endpoint's support for
-[blocking queries](/api/features/blocking.html),
```

Comment:
I think this must have been a bad rebase. Because `https://www.consul.io/api/features/blocking.html` is the correct link.

Reply:
There are multiple changes like that further down.